clusterloader2/testing/l4ilb/config.yaml

```diff
@@ -12,6 +12,9 @@
 {{$ilbWaitTimeout := DefaultParam .CL2_ILB_WAIT_TIMEOUT "10m"}}
 {{$namespaces := 1}}
 
+# Command to be executed
+{{$EXEC_COMMAND := DefaultParam .CL2_EXEC_COMMAND nil}}
```

Comment:
Rest of the test assumes that this EXEC_COMMAND will be a nodepool upgrade command, right? Can we just make the name more expliciit too? "NODEPOOL_UPGRADE_COMMAND"?

Reply:
Yes, done.